### ISSUE TITLE
Text selection handles should not be focusable in talkback.

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#76634]
```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  static const String _title = 'Flutter Code Sample';

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: _title,
      home: Scaffold(
        body: Center(
          child: TextField(),
        ),
      ),
    );
  }
}
```

1. type something into textfield
2. turn on talkback
3. double tap and long press the textfield to bring up selection handles (the blue water drop like cursor)
4. the selection handles are focusable.

I accidentally found this bug when i was triaging an internal test failure. According to accessibility guideline, any tappable component must have a label. These tappable handles failed this requirement. I double checked the native android behavior, and it looks like the selection handles can't be focused in talkback. We should hide our selection handle from the semantics, too
